stable/artifactory-jcr/values.yaml

```diff
@@ -37,6 +37,8 @@ artifactory:
 
   installer:
     platform: jcr-helm
+  
+  installerInfo: '{"productId": "Helm_artifactory-jcr/{{ .Chart.Version }}", "features": [ { "featureId": "Platform/{{ default "kubernetes" .Values.installer.platform }}"}]}'
```

Comment:
I will approve the charts because charts are correct, but in reality, this does not work.
Charts are copying installer-info to bootstrap folder, which has been disabled in JCR.
So the file never ends up at right place. Arturo was going to file a bug against JCR, not sure if he did.

Reply:
@vinayagg Yes, known permissions bug - internal fix is already in place . will be rolled out in charts as part of 7.5.0 artifactory release